tests/test_plugins_os_unix_services.py

```diff
@@ -47,10 +48,26 @@ def test_unix_services(target_unix_users, fs_unix):
         ("[Unit]\nempty_value=", 'Unit_empty_value=""'),
         ("[Unit]\nnew_lines=hello \\\nworld", 'Unit_new_lines="hello world"'),
         ("[Unit]\nnew_lines=hello \\\nworld\\\ntest", 'Unit_new_lines="hello world test"'),
-        ("[Unit]\nnew_lines=hello \\\nworld\\\ntest\\\nhelp", 'Unit_new_lines="hello world test help"'),
         ("[Unit]\ntest", 'Unit_test="None"'),
+        ("[Unit]\nnew_lines=hello \\\n#Comment\n;Comment2\nworld", 'Unit_new_lines="hello world"'),
     ],
 )
 def test_unix_systemd_parser(assignment, expected_value):
     data = parse_systemd_config(StringIO(assignment))
     assert data == expected_value
+
+
+@pytest.mark.xfail
+def test_unix_systemd_known_fail():
+    # While this should return `Hello world test help` it fails because it cannot find something
```

Comment:
`something` is a bit vague, could you make that a bit more explicit. Maybe also say how it is parsed.

Probably also good to add this case:

```
lines=hello \\
world\\

test
```
as the empty line should be ignored like the comments but is not if I understood correctly.